OTAnalytics/plugin_ui/customtkinter_gui/frame_track_plotting.py

```diff
@@ -33,14 +37,26 @@ def _get_widgets(self) -> None:
         )
 
     def _place_widgets(self) -> None:
+        self.grid_rowconfigure([i for i in range(0, len(self._layers))], weight=0)
+        self.grid_rowconfigure(len(self._layers) + 1, weight=1)
+        self.grid_columnconfigure(0, weight=0)
+        self.grid_columnconfigure(1, weight=1)
         pady = 10
-        for idx, layer in enumerate(self._layers):
-            checkbox_layer = CheckBoxLayer(master=self, layer=layer)
-            checkbox_layer.grid(
-                row=idx, column=0, padx=PADX, pady=(0, pady), sticky=STICKY
+        row = 0
+        for group in self._layers:
```

Comment:
We could also use an `for row, group in  enumerate(self._layers)` to avoid incrementing row manually.

Reply:
Good point. Missed to refactor this back to `enumerate`.